### ISSUE TITLE
Appeal plus: if no source, then not mention source on page

### ISSUE THREAD

**Issue by toffehoff** [IFRCGo/ifrcgo.github.io#53]
On the key figures there is a field "Source". When this field is empty, there is still "Source:" mentioned on the page.

**Comment by renecnielsen** [comment]
I agree that we could strip it out if no source is mentioned - but we should ALWAYS be explicit about the source! If we don't know the source, we should write that source is "Unknown".
